tests/test_services/test_analytics_service.py

```
@@ -0,0 +1,674 @@
+                skill = job_data['skills'][0]
+                skill.name = f"{common_skill_name}-{i}"
+                skill.is_blacklisted = False
+
+            db.session.commit()
+
+            analytics = analytics_service.get_skill_analytics()
+
+            # CommonSkill variants should potentially appear in skills_by_success
+            # RareSkill variants should not (below threshold)
+            success_skills = [skill['skill'] for skill in analytics['skills_by_success']]
+            rare_skills_present = any(skill_name in skill for skill in success_skills)
```

Ruff (0.13.1)

Local variable `rare_skills_present` is assigned to but never used

Remove assignment to unused variable `rare_skills_present`

(F841)